src/observers/stores/duckdb.py

```diff
@@ -40,23 +44,17 @@ def connect(cls, path: Optional[str] = None) -> "DuckDBStore":
         return cls(path=path)
 
     def _init_table(self, record: "Record") -> str:
-        table_name = f"{record.table_name}_{uuid.uuid4().hex[:8]}"
-        duckdb_schema = record.duckdb_schema.replace(record.table_name, table_name)
-        self._conn.execute(duckdb_schema)
-        self._tables.append(table_name)
-        return table_name
+        self._conn.execute(record.duckdb_schema)
+        self._tables.append(record.table_name)
```

Comment:
**Breaking Change**: We will have this table name be static (_so it's easier to know where to send data and apply migrations_). If users want to have multiple processes sending to different local databases they can also use the `path` param to have different local databases.

cc: @davidberenstein1957

Reply:
Agree, as discussed before.

Reply:
@cfahlgren1 will you randomise the local path?

Reply:
No it will stay static as well. They will need to specify different paths if they have different processes. I think that's probably better as a default than potentially having multiple duckdb paths if you rerun the same script wdyt?